youtube_dl/extractor/spankbang.py

```diff
@@ -94,3 +94,34 @@ def _real_extract(self, url):
             'formats': formats,
             'age_limit': age_limit,
         }
+
+
+class SpankBangPlaylistIE(InfoExtractor):
+    _VALID_URL = r'https?://(?:(?:www|m|[a-z]{2})\.)?spankbang\.com/(?P<id>[\da-z]+)/playlist'
+    _TEST = {
+        'url': 'https://spankbang.com/ug0k/playlist/big+ass+titties',
+        'info_dict': {
+            'id': 'ug0k',
+            'title': 'Big Ass Titties playlist',
+        },
+        'playlist_mincount': 2,
+    }
+
+    def _extract_entries(self, webpage):
+        return [
+            self.url_result(
+                'http://www.%s/%s' % ('spankbang.com', video_url),
+                SpankBangIE.ie_key())
+            for video_url in re.findall(
+                r'href="/?([\da-z-]+/playlist/[^"]+)', webpage)
```

Comment:
1. This is too broad, regex should be restricted by playlist id.
2. This captures duplicates.